docs/learningLab/introToLit/progSigning.md

```diff
@@ -1,16 +1,198 @@
+  // the signature share will be automatically returned in the HTTP response from the node
+  const sigShare = await LitActions.signEcdsa({ toSign, publicKey: "0x02e5896d70c1bc4b4844458748fe0f936c7919d7968341e391fb6d82c258192e64", sigName: "sig1" });
+};
+
+go();
+`;
+
+// you need an AuthSig to auth with the nodes
+// normally you would obtain an AuthSig by calling LitJsSdk.checkAndSignAuthMessage({chain})
+const authSig = {
+  sig: "0x2bdede6164f56a601fc17a8a78327d28b54e87cf3fa20373fca1d73b804566736d76efe2dd79a4627870a50e66e1a9050ca333b6f98d9415d8bca424980611ca1c",
+  derivedVia: "web3.eth.personal.sign",
+  signedMessage:
+    "localhost wants you to sign in with your Ethereum account:\n0x9D1a5EC58232A894eBFcB5e466E3075b23101B89\n\nThis is a key for Partiful\n\nURI: https://localhost/login\nVersion: 1\nChain ID: 1\nNonce: 1LF00rraLO4f7ZSIt\nIssued At: 2022-06-03T05:59:09.959Z",
+  address: "0x9D1a5EC58232A894eBFcB5e466E3075b23101B89",
+};
+
+const runLitAction = async () => {
+  const litNodeClient = new LitJsSdk.LitNodeClient({
+    litNetwork: "serrano",
+  });
+  await litNodeClient.connect();
+  const signatures = await litNodeClient.executeJs({
+    code: litActionCode,
+    authSig,
+    jsParams: {
+      conditions: [
+        {
+          conditionType: "evmBasic",
+          contractAddress: "",
+          standardContractType: "",
+          chain: "ethereum",
+          method: "eth_getBalance",
+          parameters: [":userAddress", "latest"],
+          returnValueTest: {
+            comparator: ">=",
+            value: "1",
+          },
+        },
+      ],
+      authSig: {
+        sig: "0x2bdede6164f56a601fc17a8a78327d28b54e87cf3fa20373fca1d73b804566736d76efe2dd79a4627870a50e66e1a9050ca333b6f98d9415d8bca424980611ca1c",
+        derivedVia: "web3.eth.personal.sign",
+        signedMessage:
+          "localhost wants you to sign in with your Ethereum account:\n0x9D1a5EC58232A894eBFcB5e466E3075b23101B89\n\nThis is a key for Partiful\n\nURI: https://localhost/login\nVersion: 1\nChain ID: 1\nNonce: 1LF00rraLO4f7ZSIt\nIssued At: 2022-06-03T05:59:09.959Z",
+        address: "0x9D1a5EC58232A894eBFcB5e466E3075b23101B89",
+      },
+      chain: "ethereum",
+    },
+  });
+  console.log("signatures: ", signatures);
+};
+
+runLitAction();
+```
+
+## Use Cases
+
+Automation and interoperability to the dWeb with Lit Actions and PKPs.
+
+Below you will find some example projects and other potential ideas and use cases.
+
+**DeFi Automation**
+
+Use PKPs and Actions to automate your interactions across decentralized finance.
+
+- Condition-based transactions (ex. on-chain limit orders).
+- Recurring payments.
+- Automated vault applications for seamlessly trading asset bundles.
+
+Example Projects:
+
+- [Sling Protocol](https://github.com/Sling-Protocol/pkp-dex-sdk): An SDK for automating transactions on popular DEXs. Currently supports Uniswap V3 and 1inch.
+- [Cask](https://www.cask.fi/): Automated, recurring payments.
```

Comment:
Cask is using Lit for AC (not PKPs) .. I see that this needs to be updated in the use cases section as well.